TypeCobol.Test/ThirdParty.cs

```diff
@@ -7,6 +7,9 @@ namespace TypeCobol.Test {
 	public class ThirdParty
     {
         [TestMethod]
+#if EUROINFO_RULES
+        [Ignore]
+#endif
```

Comment:
@smedilol, do you agree that CNAF and NIST test can be skipped in EI environment ?

Reply:
No. These tests are important. We must be sure they works also with EI environment.

Reply:
fixed : CNAF and NIST tests are not skipped in EI environment
(warning diagnostics are not considered in the result of the tests)
https://github.com/TypeCobolTeam/TypeCobol/tree/2fb9ef3a95f4ddd4e357fd2dcf04ee40b556b77b